test/common_test/basic_test.cc

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright (C) 2022 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+*/
+#include <gtest/gtest.h>
+
+#include <gz/common/Filesystem.hh>
+#include <gz/common/Console.hh>
+#include <gz/common/Util.hh>
+#include <gz/plugin/Loader.hh>
+
+#include <gz/physics/FindFeatures.hh>
+#include <gz/physics/GetEntities.hh>
+#include <gz/physics/RequestEngine.hh>
+
+// The features that an engine must have to be loaded by this loader.
+using Features = gz::physics::FeatureList<
+  gz::physics::GetEngineInfo
+>;
+
+class EntityManagementFeaturesTest:
+  public testing::Test,
+  public testing::WithParamInterface<const char *>
+{
+  // Documentation inherited
+  public: void SetUp() override
+  {
+    gz::common::Console::SetVerbosity(4);
+
+    std::string libToTest;
+    if (!gz::common::env("LIB_TO_TEST", libToTest))
+    {
+      FAIL();
+    }
+
+    auto plugins = loader.LoadLib(libToTest);
+
+    pluginNames = gz::physics::FindFeatures3d<Features>::From(loader);
```

Comment:
future todo: we should also run the `3f`, `2d`, and `2f` variants of `FindFeatures`